src/server.ts

```diff
@@ -13,3 +18,27 @@ app.get("/", async (req: Request, res: Response): Promise<Response> => {
 app.listen(PORT, () => {
   console.log(`Server is running on port ${PORT}`);
 });
+
+const options = {
```

Comment:
```suggestion
const options = {
```

I guess this options should be in it's separate file,  that will look way more cleaner